main.tf

```diff
@@ -83,33 +85,36 @@ module "dcos-infrastructure" {
 /////////////////////////////////////////
 
 module "dcos-install" {
-  source = "github.com/dcos-terraform/terraform-null-dcos-install-remote-exec?ref=conditional-bootstrap"
-
   #source  = "dcos-terraform/dcos-install-remote-exec/null"
-  #version = "~> 0.0"
-
+  #version = "~> 0.1.1"
+  
+  source = "github.com/dcos-terraform/terraform-null-dcos-install-remote-exec?ref=conditional-bootstrap" 
```

Comment:
we need to update the source back to the original

Reply:
good catch indeed!

Reply:
Yes am aware. Was attempting a test and needed to revert to this original source. Reverting the revert now :)